package.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "hmpo-logger",
-  "version": "7.0.2",
+  "version": "7.0.3",
   "description": "Consistent logging for hmpo apps",
   "main": "index.js",
   "scripts": {
     "test": "npm run lint && npm run unit && npm run check-coverage && npm audit --production",
     "lint": "eslint .",
-    "unit": "nyc --reporter=lcov --reporter=text-summary _mocha test/spec/ --recursive --require test/helper",
-    "check-coverage": "nyc check-coverage"
+    "unit": "nyc --reporter=lcov --reporter=text-summary _mocha test/spec/ --recursive --require test/helper.js",
+    "check-coverage": "nyc check-coverage",
+    "start": "node app.js",
+    "pretest": "npm run test:lint",
+    "posttest": "npm audit --production",
+    "test:unit": "nyc mocha ./test/unit --recursive --require ./test/unit/helper",
+    "test:lint": "eslint . && hmpo-lintapostrophes"
```

Comment:
We can remove these, presumably installing hmpo-node-standards caused these scripts to be added but we already have a test setup here which these will conflict with